pandas/core/computation/eval.py

```diff
@@ -159,16 +160,16 @@ def _check_for_locals(expr: str, stack_level: int, parser: str):
 
 def eval(
     expr,
-    parser="pandas",
+    parser: str = "pandas",
     engine: Optional[str] = None,
-    truediv=no_default,
-    local_dict=None,
-    global_dict=None,
+    truediv: Optional[bool] = no_default,
```

Comment:
Optional[bool] doesnt fit no_default

Reply:
thanks - will leave unannotated in this PR